tools/bazar/fields/UserField.php

```diff
@@ -181,7 +181,8 @@ public function formatValuesBeforeSave($entry)
             // Do not send mails if we are importing
             // TODO improve import detection
             if (!$isImport) {
-                $mailer->notifyNewUser($wikiName, $entry[$this->emailField]);
+                if ($wiki->GetConfigValue ("signup_mail_activation") !== "1") // Do not notify the user now if we are waiting for an account activation by mail
```

Comment:
```suggestion
                $params = $this->getService(\Symfony\Component\DependencyInjection\ParameterBag\ParameterBagInterface::class);
                if (!$params->has("signup_mail_activation") || $params->get("signup_mail_activation") !== "1") // Do not notify the user now if we are waiting for an account activation by mail
```

Il est préférable d'utiliser la syntaxe que je propose pour rester au maximum compatible de la future Ectoplasme et éviter une dette technique qu'il nous faudra revenir corriger au passage à Ectoplasme.

En effet, `GetConfigValue` utilise `wiki->config` qui est un tableau facile à modifier pendant l'exécution alors qu'en passant par le `ParameterBagInterface`, une fois le YesWiki lancé, il n'est pas possible de modifier le contenu de params et ceci évite les comportements où une autre partie de YesWiki modifie la valeurs en toute discrétion